docs/getting-started/bpfctl-guide.md

```diff
@@ -86,27 +86,24 @@ There are a common set of attributes, and those MUST come before the program typ
 bpfctl load-from-file --help
 Load an eBPF program from a local .o file
 
-Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --section-name <SECTION_NAME> <COMMAND>
+Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --name <NAME> <COMMAND>
```

Comment:
```suggestion
Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --name <PROGRAM_NAME> <COMMAND>
```

Reply:
I copied my update directly from the output of the command.  I just double checked, and what I have is what bpfctl says.

Reply:
Ok that's alright.

We probably need to fix this to say `Program_name` in the documentation just to help avoid confusion.